ui/app/components/configure-aws-secret.hbs

```diff
@@ -0,0 +1,141 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+<Hds::Tabs class="has-top-margin-l" as |T|>
+  <T.Tab data-test-tab="access-to-aws">
+    Dynamic IAM root credentials
+  </T.Tab>
+  <T.Tab data-test-tab="lease">
```

Comment:
one thing to note -- this page was previously using query params to keep track of which tabs were selected. if this is helpful for ensuring users can copy/paste a URL with a tab pre-selected i can totally add that back in, just wanted to make sure it was necessary first. 🙂

Reply:
I _think_ the new designs Yankun is working on will remove tabs all together. That aside, I think removing queryParams here is okay because given the fact this page never properly transitioned after save I suspect it's not being used as much as we'd hope.

Reply:
ah, good to know! in that case i'll finish removing the query params here completely, as there are still a couple code references left to them.

Reply:
Two thoughts on the transition:
1. we make a pr that just fixes the transition and can be backported (so it couldn't depend on any changes here).
2. we address the transition here or a future pr and don't backport it. Nobody has complained.

I think ideally, we'd do number 1, but only if it was easy to both fix and backport—otherwise, we can proceed with a fix in this aws 1.18.x work.

Reply:
good point, let's do #1. thanks!